Documentation/gettingstarted/external-workloads.rst

```diff
@@ -306,14 +193,15 @@ You should see something like:
 
 .. parsed-literal::
 
-    etcd: 1/1 connected, lease-ID=7c02748328e75f57, lock lease-ID=7c02748328e75f59, has-quorum=true: 192.168.36.11:32379 - 3.4.13 (Leader)
+    KVStore:     Ok   etcd: 1/1 connected, lease-ID=7c02748328e75f57, lock lease-ID=7c02748328e75f59, has-quorum=true: https://clustermesh-apiserver.cilium.io:32379 - 3.4.13 (Leader)
+    Kubernetes:  Disabled
     ...
 
 Check that cluster DNS works:
 
 .. code:: bash
 
-    nslookup clustermesh-apiserver.kube-system.svc.cluster.local
+    nslookup -norecurse clustermesh-apiserver.kube-system.svc.cluster.local
```

Comment:
Same remark for the `.. code:: bash`, should also be a  code block for shell session.